Dockerfile

```diff
@@ -19,6 +18,9 @@ RUN wget -O- https://runtime.fivem.net/artifacts/fivem/build_proot_linux/master/
     \
  && apk -p $PWD add tini
 
+RUN FIVEM_NUM=$(echo "${FIVEM_VER}" | cut -d '-' -f 1)
+ENV FIVEM_NUM=${FIVEM_NUM}
```

Comment:
This breaks the labelling on line 40/42:
`"org.label-schema.version": ""`
Is there any way we can do this in the Dockefile without the `RUN` line to keep that?

Reply:
Strange it's working for me.
No you can only use RUN on builder

Reply:
> Strange it's working for me.

If you run `docker inspect` on the built image do you see the label?

Reply:
![image](https://github.com/user-attachments/assets/178feade-e4e3-4d0e-838b-8866a08a0e6b)
You're right, it doesn't register